packages/ingest/src/captions/getCaptionsByFileName.ts

```diff
@@ -1,7 +1,12 @@
 import { Storage } from "@google-cloud/storage";
 import { aiLogger } from "@oakai/logger";
 import type { Cue } from "webvtt-parser";
-import { WebVTTParser } from "webvtt-parser";
+import * as WebVTTModule from "webvtt-parser";
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-expect-error
+const WebVTTParser = WebVTTModule.default
+  .WebVTTParser as typeof WebVTTModule.WebVTTParser;
```

Comment:
Is this because WebVTTParser isn't typed?

Reply:
So I think `ts-node` was no longer running scripts in this package after some wider repo changes relating to ESM. So I switched to `tsx`. However, tsx failed to recognise the named exports from the vtt parser at runtime despite Typescript compiler not complaining about it. This hack was the only way I found to get tsx to run the script